_posts/2016-03-07-2016-forman-survey-analysis.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title: 2016 Foreman Survey Analysis
+date: 2016-03-07 14:34:44
+author: Greg Sutcliffe
+tags:
+- usage
+- foreman
+- events
+- plugins
+- community
+- survey
+- API
+- virtualization
+- cfgmgmtcamp
+- puppet
+- devconf
+- fosdem
+- chef
+- use cases
+---
+
+Over the course of Febrary, we again ran the Foreman Community Survey to get
+feedback on how we're doing and what needs to be addressed. I've now had some
+time to sit down and look through the data, and share some insights with you
+all.
+
+Firstly - **thank you** to all those who filled out the survey. It was a little
+bit longer this year, as we looked to get data on how we can grow the
+contributor community as well as the user community, so thanks for taking the
+time. It's great to hear all your feedback.
+
+<!--more-->
+
+## <a name="intro"></a>Contents
+
+* [Intro](/2016/03/2016-forman-survey-analysis.html/#intro)
+* [Community Diversity](/2016/03/2016-forman-survey-analysis.html/#diversity)
+* [Support and Docs](/2016/03/2016-forman-survey-analysis.html/#support)
+* [Specific Features](/2016/03/2016-forman-survey-analysis.html/#foreman-features)
+* [Upcoming Work](/2016/03/2016-forman-survey-analysis.html/#upcoming-work)
+* [Events](/2016/03/2016-forman-survey-analysis.html/#events)
+* [Contributor Survey](/2016/03/2016-forman-survey-analysis.html/#contributor-survey)
+* [Final thoughts](/2016/03/2016-forman-survey-analysis.html/#final-thoughts)
+
+Sadly, response were slightly down from last year (**76 responses**) and the
```

Comment:
Not every person answered every question, and one or two were added shortly after the survey went live.